### PR TITLE
Minor style fixes

### PR DESCRIPTION
**Pull request by tslocum**

### CODE REVIEW

**Review by hajimehoshi — COMMENTED** [review]
Could you separate this change into two PRs: fixing misspellings, and just style changes?

**Comment by tslocum** [comment]
Sure, I've opened a separate PR.

File: internal/atlas/image.go [hunk 1]
```diff
@@ -574,9 +574,9 @@ func (img *Image) Pixels(x, y, width, height int) ([]byte, error) {
 
 	bs := make([]byte, 4*width*height)
 	idx := 0
-	for j := y; j < y+height; j++ {
-		for i := x; i < x+width; i++ {
-			r, g, b, a, err := img.at(i, j)
+	for k := y; k < y+height; k++ {
+		for j := x; j < x+width; j++ {
+			r, g, b, a, err := i.at(j, k)
```
Comment (hajimehoshi) [inline internal/atlas/image.go:579]: I think (i, j) for coordinates is more conventional.

**Review by hajimehoshi — COMMENTED** [review]
I'd like to know the policy of these style changes. Could you explain why you thought these are needed? Thanks!

**Comment by tslocum** [comment]
These changes are recommended by `go vet`.  This is the command I ran:

```
go vet -composites=false ./...
```

I focused only on inconsistent receiver names and unnecessary `else` blocks (because the prior condition returns anyway).

**Comment by hajimehoshi** [comment]
Hmm, `go vet -composites=false ./...` doesn't say anything on my environment (macOS). What kind of messages did you see in what environment?

**Comment by tslocum** [comment]
Thanks, I missed that I actually executed `golang.org/x/lint/golint` (which is now deprecated, so its output might not be entirely helpful/correct).  Leaving this open just in case these changes still seem useful.

**Comment by hajimehoshi** [comment]
Thank you for confirming. IMO

* Using early returns are fine. The more early returns, the merrier.
* Renaming the variables could be controversial. I'd want to discuss more if you want to merge it.

**Comment by tslocum** [comment]
That makes sense.  I am not particularly opinionated, so I have removed the rename changes.

**Review by hajimehoshi — COMMENTED** [review]
LGTM, thanks!

**Review by hajimehoshi — APPROVED** [review]

**Pull request merged**